unittests/test.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<espa_metadata version="1.3"
-xmlns="http://espa.cr.usgs.gov/v1"
+<espa_metadata version="2.0"
```

Comment:
Is there some way to have a central location for the version number, so various code and data files don't need to be updated with that number, or some automated procedure can be used to update the version number in the data files and code?